### PR TITLE
Fixes #37132 - Fix sorting problem in os integration test

### PR DESCRIPTION
**Pull request by dosas**

### CODE REVIEW

**Comment by evgeni** [comment]
ok to test

File: test/integration/operatingsystem_js_test.rb [hunk 1]
```diff
@@ -25,8 +27,6 @@ class OperatingsystemJSTest < IntegrationTestWithJavascript
     confirm_button.click
     assert has_no_css?("#app-confirm-modal")
 
-    first_row = find("table > tbody > tr:nth-child(1)")
-    assert_not_equal os.title, first_row.find("td:nth-child(1)").text.strip
-    refute Operatingsystem.find_by_name(os.name).present?
```
Comment (ekohl) [inline test/integration/operatingsystem_js_test.rb:30]: Shouldn't we keep this assertion?

**Comment by dosas** [comment]
@evgeni looks like there is a general problem with integration tests and chrome at the moment?

**Comment by evgeni** [comment]
Yeah, GitHub updated to Chrome 121 and that broke things :(

File: test/integration/operatingsystem_js_test.rb (hunk 1 above)
Comment (dosas) [reply test/integration/operatingsystem_js_test.rb:30]: Added assertion

**Review by ekohl — APPROVED** [review]
Thanks!

**Pull request merged**